### PR TITLE
Simplified README

### PR DESCRIPTION
**Pull request by torfjelde**
The current README is from a time before we had documentation for Bijectors.jl. Now that we do, I think we should just simplify the it, and defer everything to the documentation.

### CODE REVIEW

**Comment by torfjelde** [comment]
@yebai or @devmotion happy with this?

File: README.md [hunk 1]
```diff
@@ -1,484 +1,13 @@
 # Bijectors.jl
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://turinglang.github.io/Bijectors.jl/stable)
+[![Latest](https://img.shields.io/badge/docs-latest-blue.svg)](https://turinglang.github.io/Bijectors.jl)
```
Comment (devmotion) [inline README.md:3]: Maybe keep a link to the latest release (stable) and the latest (possibly unreleased) commit (dev)?

File: README.md [hunk 2]
```diff
@@ -1,484 +1,13 @@
...
-
-
-# Bibliography
-1. Rezende, D. J., & Mohamed, S. (2015). Variational Inference With Normalizing Flows. [arXiv:1505.05770](https://arxiv.org/abs/1505.05770v6).
-2. Kucukelbir, A., Tran, D., Ranganath, R., Gelman, A., & Blei, D. M. (2016). Automatic Differentiation Variational Inference. [arXiv:1603.00788](https://arxiv.org/abs/1603.00788v1).
+If you feel you have some relevant skills and are interested in contributing, please get in touch! You can find us in the #turing channel on the [Julia Slack](https://julialang.org/slack/) or [Discourse](discourse.julialang.org). If you're having any problems, please open a Github issue, even if the problem seems small (like help figuring out an error message). Every issue you open helps us improve the library!
```
Comment (devmotion) [inline README.md:13]:
```suggestion
If you feel you have some relevant skills and are interested in contributing, please get in touch! You can find us in the #turing channel on the [Julia Slack](https://julialang.org/slack/) or [Discourse](discourse.julialang.org). If you're having any problems, please open a Github issue, even if the problem seems small (like help figuring out an error message). Every issue you open helps us to improve the library!
```

**Review by devmotion — COMMENTED** [review]
Looks good. Maybe one could also add a link to the docs in the text, but I think it's fine with the badge(s) as well.

File: README.md (hunk 1 above)
Comment (torfjelde) [reply README.md:3]: Unfortunately `stable` docs doesn't exist :confused:

File: README.md [hunk 3]
```diff
@@ -1,484 +1,13 @@
...
-1. `RealDistribution`: `Union{Cauchy, Gumbel, Laplace, Logistic, NoncentralT, Normal, NormalCanon, TDist}`,
-2. `PositiveDistribution`: `Union{BetaPrime, Chi, Chisq, Erlang, Exponential, FDist, Frechet, Gamma, InverseGamma, InverseGaussian, Kolmogorov, LogNormal, NoncentralChisq, NoncentralF, Rayleigh, Weibull}`,
-3. `UnitDistribution`: `Union{Beta, KSOneSided, NoncentralBeta}`,
-4. `SimplexDistribution`: `Union{Dirichlet}`,
-5. `PDMatDistribution`: `Union{InverseWishart, Wishart}`, and
-6. `TransformDistribution`: `Union{T, Truncated{T}} where T<:ContinuousUnivariateDistribution`.
+*A package for transforming distributions, used by [Turing.jl](https://github.com/TuringLang/Turing.jl).*
 
-All exported names from the [Distributions.jl](https://github.com/TuringLang/Bijectors.jl) package are reexported from `Bijectors`.
+Bijectors.jl implements both an interface for transforming distributions from Distributions.jl and many transformations needed in this context. This package is used heavily in the probabiistic programing language Turing.jl.
```
Comment (torfjelde) [inline README.md:9]:
```suggestion
Bijectors.jl implements both an interface for transforming distributions from Distributions.jl and many transformations needed in this context. This package is used heavily in the probabiistic programing language Turing.jl.

See the [documentation](https://turinglang.github.io/Bijectors.jl) for more.
```

**Comment by torfjelde** [comment]
Done:)

File: README.md (hunk 1 above)
Comment (devmotion) [reply README.md:3]: AFAICT the problem is that TagBot.yml contains a bug - the action does not have sufficient permissions to trigger other Github actions, and hence docs won't be built when a new tag/release is made.

Comment (devmotion) [reply README.md:3]: We should be able to fix this and then we could point to stable docs as well.

Comment (torfjelde) [reply README.md:3]: Still not finding any stable docs :confused:

Comment (devmotion) [reply README.md:3]: Was a new release created after the TagBot PR was merged? I think only then a stable docs build should be triggered.

Comment (torfjelde) [reply README.md:3]: Aaah, gotcha. Aight, I'll add it to the README then.

**Comment by torfjelde** [comment]
Good to go now @devmotion ?

**Review by devmotion — APPROVED** [review]

**Pull request merged**